specs/consensus.md

```diff
@@ -46,6 +47,7 @@ Consensus Rules
 | name                                 | type     | value   | unit    | description                                                                                                                                                         |
 | ------------------------------------ | -------- | ------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `AVAILABLE_DATA_ORIGINAL_SQUARE_MAX` | `uint64` |         | `share` | Maximum number of rows/columns of the original data [shares](data_structures.md#share) in [square layout](data_structures.md#arranging-available-data-into-shares). |
+| `BLOCK_TIME`                         | `uint64` |         | second  | Block time, in seconds.                                                                                                                                             |
```

Comment:
In the implementation this will be controlled by the consensus timeouts, right? 
https://github.com/tendermint/tendermint/blob/e3b96107af324fc6a7761563c4cf58d6c4119ad6/config/toml.go#L387-L402

Reply:
Yes.